ftw/noticeboard/content/notice.py

```diff
@@ -20,7 +20,8 @@ class INotice(Interface):
 
 def user_mail():
     if not api.user.is_anonymous():
-        return safe_unicode(api.user.get_current().getProperty('email'))
+        email = api.user.get_current().getProperty('email')
+        return safe_unicode(email) or u''
```

Comment:
If it is `None`, wouldn't safe_unicode break before it goes to the `or` case?

Reply:
No it unfortunately still returns `None`, but we need a unicode string for sure